src/main/kotlin/app/revanced/patches/youtube/layout/theme/patch/ThemePatch.kt

```diff
@@ -0,0 +1,51 @@
+package app.revanced.patches.youtube.layout.theme.patch
+
+import app.revanced.patcher.annotation.*
+import app.revanced.patcher.data.impl.ResourceData
+import app.revanced.patcher.patch.OptionsContainer
+import app.revanced.patcher.patch.PatchOption
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.DependsOn
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patcher.patch.impl.ResourcePatch
+import app.revanced.patches.youtube.layout.theme.Themes
+import app.revanced.patches.youtube.misc.manifest.patch.FixLocaleConfigErrorPatch
+import org.w3c.dom.Element
+
+@Patch
+@DependsOn([FixLocaleConfigErrorPatch::class])
+@Name("theme")
+@Description("Enables a custom theme.")
+@Compatibility([Package("com.google.android.youtube")])
```

Comment:
While using the annotation directly here works, it would be better to be consistent with the other patches. See `AmoledCompatibility`.

Reply:
I did it because it's not shared, but for consistency I can understand why. I'll do that then

Reply:
Yeah theres multiple patches where a separate annotation is technically not needed, it's soley for consistency.